src/inventory.rs

```diff
@@ -261,6 +260,7 @@ impl DeploymentInventoryService {
             let sn_auditor_version = &auditor_node_registry.auditor.as_ref().unwrap().version;
 
             BinaryOption::Versioned {
+                safe_version: "0.0.1".parse()?, // todo: store safe version in the safenodeman registry?
```

Comment:
I think we should probably store this somewhere on the genesis node and fetch it with an additional simple playbook. I can't really see how it would make sense for it to be in the node manager's registry.